api/src/main/java/com/epam/pipeline/manager/user/UserApiService.java

```diff
@@ -66,6 +67,22 @@ public PipelineUser updateUserBlockingStatus(final Long id, final boolean blockS
         return userManager.updateUserBlockingStatus(id, blockStatus);
     }
 
+    /**
+     * Updates `blocked` field of given group or add new entity to a database if no such exists
```

Comment:
Please, update javadoc with actual behavior: this method updates or creates a new one